variables.tf

```diff
@@ -683,6 +685,8 @@ variable "runner_worker_docker_machine_instance" {
     subnet_ids                 = optional(list(string), [])
     types                      = optional(list(string), ["m5.large"])
     volume_type                = optional(string, "gp2")
+    volume_throughput          = optional(number, 125)
+    volume_iops                = optional(number, 3000)
```

Comment:
question: These are the default values today? I am not able to check this in my environment now.

Reply:
Simply looking at our current volumes: Yes

AFAIK these are the gp3 defaults by AWS

![Screenshot 2024-01-15 at 11 12 23](https://github.com/cattle-ops/terraform-aws-gitlab-runner/assets/25439373/38336dca-54b9-466d-afc7-5f54f65ed212)